readme.md

```diff
@@ -48,20 +48,42 @@ dust({
 	}
 });
 ```
+##### config
+
+Type: `object`  
+Default: `{ whitespace: false, amd: false, cjs: false }`
+
+Corresponds to `dust.config`.  Use it to override any dust configuration value.
```

Comment:
Nope, I can't find any such documentation.  I've found references to individual `dust.config` values in release notes and inside the guides at dustjs.com.